edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Edgelet/versioning/ModuleManagementHttpClientVersioned.cs

```diff
@@ -41,6 +45,7 @@ abstract class ModuleManagementHttpClientVersioned
             this.Version = Preconditions.CheckNotNull(version, nameof(version));
             this.transientErrorDetectionStrategy = transientErrorDetectionStrategy;
             this.operationTimeout = operationTimeout.GetOrElse(DefaultOperationTimeout);
+            this.failedSocketCount = 0;
```

Comment:
Not needed; already set to 0 in line 36.

Reply:
I'm paranoid.